include/zephyr/drivers/cellular.h

```diff
@@ -92,6 +92,8 @@ enum cellular_event {
 	CELLULAR_EVENT_MODEM_INFO_CHANGED = BIT(0),
 	/** Cellular network registration status changed */
 	CELLULAR_EVENT_REGISTRATION_STATUS_CHANGED = BIT(1),
+	/** Periodic script result */
+	CELLULAR_EVENT_PERIODIC_SCRIPT_RESULT = BIT(2),
```

Comment:
This event is internal to the modem_cellular.c driver, unlike `CELLULAR_EVENT_REGISTRATION_STATUS_CHANGED` which is generic to any modem driver. I believe it would be more portable to make this event generic to what it indicates, rather than what it "is". Something like `CELLULAR_EVENT_MODEM_UNRESPONSIVE`, or `CELLULAR_EVENT_MODEM_DEAD`.

Reply:
The event is called on both success and failure so those names don't work, but I see your point. Would you be happy with `CELLULAR_EVENT_MODEM_KEEPALIVE`?